sysdig/resource_sysdig_secure_cloud_auth_account.go

```diff
@@ -33,6 +33,10 @@ var (
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			SchemaVersion: {
```

Comment:
@cgeers I believe we will need to update the component resource as well with this change.

Reply:
good call. Yes, this already serves the immediate intended purpose, but there's no reason to leave this half implemented. I've added added the conversion steps to the component resourced as well.